### PR TITLE
Sign up, Sign In forms

### PR DESCRIPTION
**Pull request by eapenzacharias**
## Sign up, Sign In forms

- [x] Create Sign up form
- [x] Create Sign in form
- [x] Connected Landing page buttons to the form pages
- [x] Added React Router Dom

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
 function App() {
   return (
     <>
       <SideNav />
-      <LandingView />
+      <Routes>
+        <Route path="/" element={<LandingView />} />
+        <Route path="signin" element={<SignInForm />} />
+        <Route path="signup" element={<SignUpForm />} />
+      </Routes>
```
Comment (Maha-Magdy) [inline src/App.js:15]: This is a nice start you have done so far to build routes @eapenzacharias ✌,

File: src/assets/stylesheets/index.css [hunk 2]
```diff
@@ -3,6 +3,7 @@
 @tailwind utilities;
 
 body {
+  background-color: #b2dfff;
```
Comment (Maha-Magdy) [inline src/assets/stylesheets/index.css]: why are you adding this color as the default background for all pages, some pages will remain white as per the design given?

File: tailwind.config.js [hunk 3]
```diff
@@ -3,7 +3,12 @@ module.exports = {
     './src/**/*.{js,jsx,ts,tsx}',
   ],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        background: '#b2dfff',
+        darkblue: '#304656',
```
Comment (Maha-Magdy) [inline tailwind.config.js:9]: Thanks for adding this 😇 This will save time.

**Review by Maha-Magdy — COMMENTED** [review]
Hey Eapen (●'◡'●)

You have done great work to initialize the basic forms in the application so far. ✨ 🎉
I have focused more on the FE part as I will wait for the coming PR to review the configuration to connect them to the BE as we agreed.

Thanks ✌😇

File: src/assets/stylesheets/index.css (hunk 2 above)
Comment (eapenzacharias) [reply src/assets/stylesheets/index.css]: I will create another div with this bg color instead of giving it directly to the body before merging

**Pull request merged**